templates/postgis/11-2.5/Dockerfile

```diff
@@ -23,40 +59,71 @@ RUN set -ex \
     && rm postgis.tar.gz \
     \
     && apk add --no-cache --virtual .build-deps \
+        \
+        gdal-dev~=${GDAL_ALPINE_VER} \
+        geos-dev~=${GEOS_ALPINE_VER} \
+        proj-dev~=${PROJ_ALPINE_VER} \
+        \
         autoconf \
         automake \
+        clang-dev \
         file \
+        g++ \
+        gcc \
+        gettext-dev \
         json-c-dev \
         libtool \
         libxml2-dev \
+        llvm-dev \
         make \
+        pcre-dev \
         perl \
-        clang-dev \
-        g++ \
-        gcc \
-        gdal-dev \
-        geos-dev \
-        llvm10-dev \
-        proj-dev \
         protobuf-c-dev \
+    \
+# build PostGIS
+    \
     && cd /usr/src/postgis \
+    && gettextize \
     && ./autogen.sh \
-# configure options taken from:
-# https://anonscm.debian.org/cgit/pkg-grass/postgis.git/tree/debian/rules?h=jessie
     && ./configure \
-#       --with-gui \
-    && make \
+        --with-pcredir="$(pcre-config --prefix)" \
+    && make -j$(nproc) \
     && make install \
+    \
+# regress check
+    && mkdir /tempdb \
+    && chown -R postgres:postgres /tempdb \
+    && su postgres -c 'pg_ctl -D /tempdb init' \
+    && su postgres -c 'pg_ctl -D /tempdb start' \
+    && cd regress \
+    && make -j$(nproc) check RUNTESTFLAGS=--extension   PGUSER=postgres \
+    #&& make -j$(nproc) check RUNTESTFLAGS=--dumprestore PGUSER=postgres \
+    #&& make garden                                      PGUSER=postgres \
+    \
+    && su postgres -c 'psql    -c "CREATE EXTENSION IF NOT EXISTS postgis;"' \
+    && su postgres -c 'psql -t -c "SELECT version();"'              >> /_pgis_full_version.txt \
+    && su postgres -c 'psql -t -c "SELECT PostGIS_Full_Version();"' >> /_pgis_full_version.txt \
+    \
+    && su postgres -c 'pg_ctl -D /tempdb --mode=immediate stop' \
+    && rm -rf /tempdb \
+    && rm -rf /tmp/pgis_reg \
+# add .postgis-rundeps
     && apk add --no-cache --virtual .postgis-rundeps \
+        \
+        gdal~=${GDAL_ALPINE_VER} \
+        geos~=${GEOS_ALPINE_VER} \
+        proj~=${PROJ_ALPINE_VER} \
+        \
         json-c \
-        geos \
-        gdal \
-        proj \
         libstdc++ \
+        pcre \
         protobuf-c \
+# clean
     && cd / \
     && rm -rf /usr/src/postgis \
-    && apk del .fetch-deps .build-deps
+    && apk del .fetch-deps .build-deps \
+# print PostGIS_Full_Version() for the log. ( experimental & internal )
+    && cat /_pgis_full_version.txt
 
-COPY ./initdb-postgis.sh /docker-entrypoint-initdb.d/postgis.sh
+COPY ./initdb-postgis.sh /docker-entrypoint-initdb.d/10_postgis.sh
```

Comment:
should this be 11?

Reply:
Dockerfile is copy from 
https://github.com/postgis/docker-postgis/blob/master/11.6-2.5/alpine/Dockerfile
without any changes. 

Reason for rename :
> Rename init script to 10_postgis.sh, update make -j (https://github.com/postgis/docker-postgis/pull/182)
> * In order to make derived docker images more predictable,
>   rename all `/docker-entrypoint-initdb.d/postgis.sh` to `10_postgis.sh`.
>   This way if the derived image needs to run something after (or before)
>   `10_postig.sh`, it can do so by naming new script with correct number.